source/channels/message-priority.rst

```diff
@@ -28,6 +28,25 @@ To set the priority of a new message:
 
 1. Select the **Message Priority** |priority-icon| icon in the message formatting toolbar. 
 2. Select the priority for the message. Messages have a standard priority by default.
+3. Select **Apply**
+
+Send persistent notifications
+-----------------------------
+
+From Mattermost v8.0, when you add an urgent priority label, and your message @mentions at least one other user, you can enable persistent notifications which notify recipients every 5 minutes for up to 30 minutes until the recipient acknowledges or replies to the message.
+
+To enable persistent notifications for a message:
+
+1. Compose a message with at least one @mention.
+2. Select the **Message Priority** |priority-icon| icon in the message formatting toolbar. 
+3. Select **Urgent**.
+4. Select **Send persistent notifications**.
+5. Select **Apply**.
+
+.. note:: 
+
+  - Recipients must have Mattermost notifications enabled to receive persistent notifications.
```

Comment:
@esethna - I'm unable to see persistent notifications working in a configured test environment while trying to @mention myself. Is it expected that this wouldn't work?

Also - Do users need to enable acknowledgement in order for persistent notifications to work? I ask because I've read that the notifications don't stop until a recipient acknowledges or responds.